docs/_templates/quickstart.rst

```diff
@@ -0,0 +1,69 @@
+.. _quick_start:
+
+Quick Start
+===========
+
+This quick-start contains an overview of how the ``xpdAn`` software works.
+To understand more details about it, please refer to the detailed documentation in :ref:`xpdu`
+
+Please use this page as a reminder of the workflow and to copy & paste code snippets into the
+ipython terminals that are controlling your experiment and analysis.  After
+pasting, hit enter.
+
+Remember, to post questions about anything XPD, including software, and to see archived answers, at the `XPD-Users Google group
+<https://groups.google.com/forum/#!forum/xpd-users;context-place=overview>`_ . If you are not already a member please request to join
+the community
+
+XPD Data Analysis Workflow Summary
+------------------------------------
+
+This is the summary of the steps for analysis data at XPD. They are explained below.
```

Comment:
for analyzing, or for analysis of

Also What does "they" refer to?  replace with "the steps"?